auth0_flutter/android/src/main/kotlin/com/auth0/auth0_flutter/AuthenticationExceptionExtensions.kt

```diff
@@ -3,31 +3,33 @@ package com.auth0.auth0_flutter
 import com.auth0.android.authentication.AuthenticationException
 
 fun AuthenticationException.toMap(): Map<String, Any> {
-    return mapOf(
-        "_statusCode" to this.statusCode,
-        "_errorFlags" to mapOf(
-            "isMultifactorRequired" to this.isMultifactorRequired,
-            "isMultifactorEnrollRequired" to this.isMultifactorEnrollRequired,
-            "isMultifactorCodeInvalid" to this.isMultifactorCodeInvalid,
-            "isMultifactorTokenInvalid" to this.isMultifactorTokenInvalid,
-            "isPasswordNotStrongEnough" to this.isPasswordNotStrongEnough,
-            "isPasswordAlreadyUsed" to this.isPasswordAlreadyUsed,
-            "isRuleError" to this.isRuleError,
-            "isInvalidCredentials" to this.isInvalidCredentials,
-            "isRefreshTokenDeleted" to this.isRefreshTokenDeleted,
-            "isAccessDenied" to this.isAccessDenied,
-            "isTooManyAttempts" to this.isTooManyAttempts,
-            "isVerificationRequired" to this.isVerificationRequired,
-            "isNetworkError" to this.isNetworkError,
-            "isBrowserAppNotAvailable" to this.isBrowserAppNotAvailable,
-            "isPKCENotAvailable" to this.isPKCENotAvailable,
-            "isInvalidAuthorizeURL" to this.isInvalidAuthorizeURL,
-            "isInvalidConfiguration" to this.isInvalidConfiguration,
-            "isCanceled" to this.isCanceled,
-            "isPasswordLeaked" to this.isPasswordLeaked,
-            "isLoginRequired" to this.isLoginRequired,
-        )
-    )
+    var ex = this;
```

Comment:
Let's use a non-abbreviated word for better readability, e.g. `exception`.

Reply:
I'm not the most experienced with kotlin, but the `buildMap` method was my solution to have an optional key-value pair in the map, instead of changing the signature of the map from <String, Any> to <String, Any?>. In this method referencing `this` would point to the map instead of the exception, which is why I had to introduce this variable.
If you have a better method, feel free to let me know

Reply:
No worries, I'm no Kotlin expert myself either. I'll ask @poovamraj to review the Android bits once we're close to merging.